Create ui-c8y-1020-23-1-error-appears-for-checking-active-filters-on-subtenants-6904-graft-release.md (#2405)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1020-23-1-error-appears-for-checking-active-filters-on-subtenants-6904-graft-release.md

---
date: ""

> title: Fix error when checking active filters on subtenants

Reviewer: ```suggestion
title: Subtenants can now reliably check their active filters
```

> title: Subtenants can now reliably check their active filters

Reviewer: ```suggestion
title: Users can now reliably check active filters for subtenants
```

> title: error appears for checking active filters on subtenants (#6904) [GRAFT][release/cd] (#6981)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Fix error when checking active filters on subtenants
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:

>   - value: component-YbYJ3gLU_

Reviewer: ```suggestion
  - value: component-0UgqXH1Ys 
```

>     label: Web SDK

Reviewer: ```suggestion
    label: Administration
```

build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-59337
version: 1020.23.1
---

> error appears for checking active filters on subtenants (#6904) [GRAFT][release/cd] (#6981)

Reviewer 2: ```suggestion
In some situations, an error occurred when checking active filters on subtenants, preventing users from properly viewing and managing the filters. This issue has been resolved. Subtenants can now reliably check their active filters without encountering an error. This change ensures that the filtering functionality works as expected for all subtenants.
```

> error appears for checking active filters on subtenants (#6904) [GRAFT][release/cd] (#6981)

Author: The Claude AI suggested to use the following body:
```suggestion
In some situations, an error occurred when checking active filters on subtenants, preventing users from properly viewing and managing the filters. This issue has been resolved. Subtenants can now reliably check their active filters without encountering an error. This change ensures that the filter management functionality works as expected for all subtenants.
```

Reviewer 2: Resolved.

> In some situations, an error occurred when checking active filters on subtenants, preventing users from properly viewing and managing the filters. This issue has been resolved. Subtenants can now reliably check their active filters without encountering an error. This change ensures that the filtering functionality works as expected for all subtenants.

Reviewer: ```suggestion
In some situations an error occurred when checking active filters on subtenants, preventing users from properly viewing and managing the filters. This issue has been resolved. Subtenants can now reliably check their active filters without encountering an error. 
```
